v1-api-conversion/README.md

```diff
@@ -0,0 +1,369 @@
+
+#### Supporting automatic conversion
+
+The following changes can be done manually or using the conversion tool.
+
+* Change the `apiVersion` to `kafka.strimzi.io/v1`.
+* Delete the `.spec.zookeeper` section.
+  ZooKeeper-based Apache Kafka clusters are not supported anymore.
+* Delete the `.spec.jmxTrans` section.
+  JMX Trans is not supported anymore.
+* Delete the following Kafka properties which are nto supported anymore:
+  * `.spec.kafka.replicas` (replicas are configured through the `KafkaNodePool` resources)
+  * `.spec.kafka.storage` (storage is configured through the `KafkaNodePool` resources)
+  * `.spec.kafka.template.statefulset` 
+* Delete the following Cruise Control properties which are not supported anymore:
+  * `.spec.cruiseControl.tlsSidecar`
+  * `.spec.cruiseControl.template.tlsSidecarContainer`
+  * `.spec.cruiseControl.brokerCapacity.disk`
+  * `.spec.cruiseControl.brokerCapacity.cpuUtilization`
+* Delete the `.spec.kafkaExporter.template.service` property.
+* Delete the following Entity Operator properties which are not supported anymore:
+  * `.spec.entityOperator.tlsSidecar`
+  * `.spec.entityOperator.template.tlsSidecarContainer`
+  * `.spec.entityOperator.topicOperator.topicMetadataMaxAttempts`
+  * `.spec.entityOperator.topicOperator.zookeeperSessionTimeoutSeconds`
+  * `.spec.entityOperator.userOperator.zookeeperSessionTimeoutSeconds`
+* Delete the `.spec.entityOperator.topicOperator.reconciliationIntervalSeconds` property and use the `.spec.entityOperator.topicOperator.reconciliationIntervalMs` property instead.
+  The value should be the original value of `reconciliationIntervalSeconds` times 1000.
+* Delete the `.spec.entityOperator.userOperator.reconciliationIntervalSeconds` property and use the `.spec.entityOperator.userOperator.reconciliationIntervalMs` property instead.
+  The value should be the original value of `reconciliationIntervalSeconds` times 1000.
+* In YAML files that include the status section, remove the `.status.registeredNodeIds`, `.status.kafkaMetadataState`, and `.status.listeners[].type` fields.
+  This conversion is not needed for any resources that exist in your Kubernetes cluster and are managed by Strimzi Cluster Operator.
+
+### `KafkaConnect` CR
+
+#### Requiring manual conversion
+
+The following changes cannot be done by the conversion tool and have to be applied manually.
+
+* The `.spec` section is required and have to be present.
+* `type: oauth` authentication is not supported in the `v1` API.
+  Use `type: custom` authentication instead.
+* `.spec.externalConfiguration` is not supported in the `v1` API.
+  Use the `.spec.template` section instead to add custom volumes or environment variables.
+
+#### Supporting automatic conversion
+
+The following changes can be done manually or using the conversion tool.
+
+* The `.spec.replicas` field is required in the `v1` API.
+  If you do not have it in your `KafkaConnect` custom resource, add it and set it to `3`.
```

Comment:
```suggestion
  If you do not have it in your `KafkaConnect` custom resource, add it and set it to `3` (recommended).
```

Reply:
In this case, it is not the recommended value. The current default is `3`. So if the `replicas` field is not set, the user has today 3 replicas and has to set it to `3` replicas during the conversion.